packages/rulesets/src/spectral/functions/tags-are-not-allowed-for-proxy-resources.ts

```diff
@@ -0,0 +1,29 @@
+import _ from "lodash"
+import { deepFindObjectKeyPath, getProperties } from "./utils"
+
+const TAGS = "tags"
+const PROPERTIES = "properties"
+const NestedPROPERTIES = "properties"
```

Comment:
>properties [](http://example.com/codeflow?start=26&length=10)

do we need this?

Reply:
I can directly use properties while concatenating the path, but to make it more readable I defined here.

Reply:
I think this is generally not considered good coding practice as it creates redundancy